meta/main.yml

```diff
@@ -5,9 +5,14 @@ galaxy_info:
   license: Apache
   min_ansible_version: 2.4
   platforms:
-  - name: Ubuntu
-    versions:
-    - xenial
+    - name: Ubuntu
+      versions:
+        - xenial
+    - name: Windows
+      versions:
+        - "2016"
```

Comment:
I'm not sure this is a valid version - galaxy API only gave back `2012R2` for Windows, and I haven't tested with that.